### DISCUSSION TITLE
Missing test dep on anyio?

### DISCUSSION THREAD

**Discussion by mgorny** [encode/httpx#2542] [Potential Issue] [answered]
FWICS e27d1b83330d0520f87a7d1da1818e4dcb2add02 started using `anyio` in the test suite but it doesn't seem to be listed in `requirements.txt`.

CC @graingert, @tomchristie

**Comment by lovelydinosaur** [comment, marked as answer]
`anyio` is included by `httpcore`.

https://github.com/encode/httpcore/blob/master/setup.py

**Reply by mgorny** [reply]
Ok, I suppose this makes sense. Since you're pinning test dependencies, listing it explicitly could cause version conflicts.
